src/intelli/private/gui_utils.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * GTlab IntelliGraph
+ *
+ *  SPDX-License-Identifier: BSD-3-Clause
+ *  SPDX-FileCopyrightText: 2025 German Aerospace Center
+ *
+ *  Author: Marius Bröcker <marius.broecker@dlr.de>
+ */
+#ifndef GUI_UTILS_H
+#define GUI_UTILS_H
```

Comment:
please use this as a header guard
```suggestion
#ifndef GT_INTELLI_GUI_UTILS_H
#define GT_INTELLI_GUI_UTILS_H
```